extensions/3DTILES_metadata/schema/group.schema.json

```diff
@@ -10,9 +10,22 @@
         }
     ],
     "properties": {
+        "id": {
+            "type": "string",
+            "minLength": 1,
+            "description": "Unique identifier for the group."
+        },
```

Comment:
From offline discussion: Now that https://github.com/CesiumGS/3d-tiles/pull/646 is merged this section can be updated similar to `schema.schema.json`.